### PR TITLE
Fix #110936 www.nivea.de

### LINKED ISSUE #110936: www.nivea.de
### Issue URL (Annoyance)
 [https://www.nivea.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.nivea.de%2F)
### Comment
Username: @Morku
> Cookie Pop-up

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.8
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters
DNS filtering: | server: `Cloudflare DNS-over-HTTPS`<br>filters: `"AdGuard DNS filter" with AdGuard Home`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard German
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by Morku**
Fix #110936

### CODE REVIEW

**Review by Morku — COMMENTED** [review]
checked other language pages, too

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -4052,7 +4052,7 @@ decrypt.co##.iDynzw
 fialkaspb.ru##.bottom__cookie_block
 thecustomdroid.com###cookie-consent-lite
 wolframalpha.com###__next > div[class] > header ~ section[class]
-absolventa.de,porzellanhandel24.de,blastbrigade.com,blic.rs,queer.de,inqa.de,bmas.de,mail.ru###cmpbox
+nivea.sk,nivea.si,nivea.se,nivea.rs,nivea.ro,nivea.pt,nivea.pl,nivea.no,nivea.nl,nivea.lv,nivea.lt,nivea.it,nivea.gr,nivea.hr,nivea.hu,nivea.fr,nivea.fi,nivea.es,nivea.ee,nivea.dk,nivea.cz,nivea.bg,nivea.at,nivea.de,absolventa.de,porzellanhandel24.de,blastbrigade.com,blic.rs,queer.de,inqa.de,bmas.de,mail.ru###cmpbox
```
Comment (gork7777) [inline AnnoyancesFilter/sections/cookies_specific.txt:4055]: It's better use `nivea.*` rule.

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:4055]: @Morku I agree about the rule above.
```
nivea.*,absolventa.de,porzellanhandel24.de,blastbrigade.com,blic.rs,queer.de,inqa.de,bmas.de,mail.ru###cmpbox
```

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -6215,7 +6215,7 @@ orf.at##.b-dsgvo-banner
...
+nivea.com.au,nivea-ir.com,nivea-me.com,nivea.az,nivea.be,nivea.by,nivea.ca,nivea.ch,nivea.cl,nivea.co.id,nivea.co.il,nivea.co.ke,nivea.co.kr,nivea.co.th,nivea.co.uk,nivea.co.za,nivea.com.ar,nivea.com.br,nivea.com.cn,nivea.com.co,nivea.com.ec,nivea.com.gh,nivea.com.gt,nivea.com.mx,nivea.com.my,nivea.com.ng,nivea.com.pe,nivea.com.py,nivea.com.tr,nivea.com.tw,nivea.com.uy,nivea.com.ve,nivea.com.vn,nivea.ge,nivea.ie,nivea.in,nivea.kz,nivea.ma,nivea.ph,nivea.ru,nivea.ua,niveausa.com##.nx-cookie-policy-popup
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:6218]: @Morku you can also improve the rule here.
```
nivea.*##.nx-cookie-policy-popup
```

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by Morku — COMMENTED** [review]
update

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
> update

Thank you! I will be glad to see you in futher pull requests :)

**Comment by Morku** [comment]
@zloyden 
Thank you. I will try :) but my knowledge is still very limited and still don't know like when using blocking rule or extended css. Also still a lot of try and error by comparing with examples.

**Comment by gork7777** [comment]
@Morku You can also check this: [Syntax Meanings](https://github.com/DandelionSprout/adfilt/blob/master/Wiki/SyntaxMeaningsThatAreActuallyHumanReadable.md)

**Comment by zloyden** [comment]
@Morku [this ](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters) can help you in creating rules.

> still don't know like when using blocking rule or extended css

Well, in general, it will be much easier for you when you create a lot of rules.
There is a golden rule "the simpler the rule, the better it is".
We use extended css when the HTML code is not simple and / or cosmetic `##` rules may break the website.

**Pull request merged**